config/routes.rb

```diff
@@ -1,5 +1,5 @@
 Rails.application.routes.draw do
-  root to: redirect('/is_it_working')
+  root to: redirect('/status/all')
```

Comment:
I think this should be `/status` or more specifically `/status/default`. @eefahy and I have not been using the `all` endpoint for monintoring thus far.